src/Agoda.Analyzers.Test/AgodaCustom/AG0039UnitTests.cs

```diff
@@ -8,107 +8,51 @@ namespace Agoda.Analyzers.Test.AgodaCustom;
-                    ///// 
-                    ///// </summary>
-                    //public const int int2 = 1;
-                    ///// <summary>
-                    ///// 
-                    ///// </summary>
-                    //public void DoesNothing() { }
-                    ///// <summary>
-                    ///// 
-                    ///// </summary>
-                    //public event SampleEventHandler SampleEvent;
-                    ///// <summary>
-                    ///// 
-                    ///// </summary>
-                    ///// <param name=""sender""></param>
-                    //public delegate void SampleEventHandler(object sender);
-                }
-")]
-    [TestCase(@"
-                /// <include file='xml_include_tag.xml' path='MyDocs/MyMembers[@name=""test""]/*' />
-                public class NotDoc
+    public async Task AG0039_At120LinesOfCode_ShowWarning()
+    {
+        var code = @"
+                internal class InternalClass
                 {
-                    /// <include file='xml_include_tag.xml' path='MyDocs/MyMembers[@name=""test""]/*' />
-                    public string str1 { get; }
-                    /// <include file='xml_include_tag.xml' path='MyDocs/MyMembers[@name=""test""]/*' />
-                    public const int int2 = 1;
-                    /// <include file='xml_include_tag.xml' path='MyDocs/MyMembers[@name=""test""]/*' />
-                    public void DoesNothing() { }
-                    /// <include file='xml_include_tag.xml' path='MyDocs/MyMembers[@name=""test""]/*' />
-                    public event SampleEventHandler SampleEvent;
-                    /// <include file='xml_include_tag.xml' path='MyDocs/MyMembers[@name=""test""]/*' />
-                    public delegate void SampleEventHandler(object sender);
+                    private void MyMethod()
+                    {
+var a = 1;
+";
+        for (int i = 0; i <= 121; i++)
+        {
+            code += "a += 1;\n";
+        }
+        code += @"
+                    }
                 }
-")]
-    public async Task AG0039_WithDoc_ShowNoWarning(string code)
-    {
-        await VerifyDiagnosticsAsync(code, EmptyDiagnosticResults);
+";
+        await VerifyDiagnosticsAsync(code, new[]{
+            new DiagnosticLocation(4, 34, "MyMethod","126"),
+        });
     }
 
     [Test]
-    [TestCase(@"
-				public class NotDoc
-				{
-					public string str1 {get;}
-					public const int int2 = 1;
-                    public void DoesNothing() {}
-                    public event SampleEventHandler SampleEvent;
-                    public delegate void SampleEventHandler(object sender);
-                    internal string internalString;
-                    internal string internalStringProp {get;}
-                    protected string protectedString;
-                    private int _privateInt;
-				}
+    public async Task AG0039_At120LinesOfWhiteSpace_DoNotShowWarning()
```

Comment:
Same as above.
Consider testing 40 lines of code, with added empty lines as well (maybe lines that contain white-spaces as well).

Reply:
fixed